src/SPC/builder/windows/WindowsBuilder.php

```diff
@@ -109,6 +110,7 @@ public function buildPHP(int $build_target = BUILD_TARGET_NONE): void
                 ($enableCli ? '--enable-cli=yes ' : '--enable-cli=no ') .
                 ($enableMicro ? ('--enable-micro=yes ' . $micro_logo . $micro_w32) : '--enable-micro=no ') .
                 ($enableEmbed ? '--enable-embed=yes ' : '--enable-embed=no ') .
+                ($enableCgi ? '--enable-cgi=yes ' : '--enable-cgi=no ') .
```

Comment:
we should change these to be in line with the unix version

$enable ? '--enable-sapi' : '--disable-sapi'.